### ISSUE TITLE
Duplicate sessions on BrowserStack

### ISSUE THREAD

**Issue by maxbeatty** [theintern/intern#1068]
## Expected behavior

BrowserStack starts a single session per browser version with this environments configuration (from `showConfig`):

```js
  "environments": [
    {
      "browserName": "node"
    },
    {
      "browserName": "internet explorer",
      "browserVersion": [
        "10",
        "11"
      ],
      "browserstack.console": "errors",
      "browserstack.debug": true,
      "browserstack.networkLogs": true,
      "browserstack.seleniumLogs": true,
      "os_version": "7",
      "version": [
        "10",
        "11"
      ]
    }
  ],
```

1. IE 10 on Windows 7
2. IE 11 on Windows 7

## Current behavior

BrowserStack is starting two sessions per browser version with this environments configuration:

![Screen Shot 2020-01-23 at 11 00 25 AM](https://user-images.githubusercontent.com/93179/73014802-adb3fc80-3dcf-11ea-87e7-d3f1c1c4a8e5.png)

1. IE 10 on Windows 7
2. IE 10 on Windows 7
3. IE 11 on Windows 10
4. IE 11 on Windows 10

Removing "os_version" still results in 4 sessions:

![Screen Shot 2020-01-23 at 10 54 59 AM](https://user-images.githubusercontent.com/93179/73014938-e94ec680-3dcf-11ea-9794-86f50d1b0362.png)

1. IE 10 on Windows 8
2. IE 10 on Windows 8
3. IE 11 on Windows 10
4. IE 11 on Windows 7

## Possible solution

Is this BrowserStack's fault?

Stop listing versions and enumerate individually? (That's my current workaround)

## Steps to reproduce (for bugs)

Specify a list of versions instead of a single version and run against BrowserStack. This does not appear to be specific to browser (happens with IE, Edge, Safari, Firefox, and Chrome).

## Environment

Intern version: 4.8.0
Node version: 13.5.0
NPM version: 6.13.4
Browser version: BrowserStack

**Comment by jason0x43** [comment]
This is an environment handling bug in Intern (#1064). It's been fixed in master. This issue can stay open until the fix has been backported to 4.8.

**Comment by jason0x43** [comment]
Reminder: the fix in master is at 85101e90a

**Comment by jason0x43** [comment]
Added to 4.x in 3ce5645
